java-manta-client/src/main/java/com/joyent/manta/client/crypto/EncryptingPartEntity.java

```diff
@@ -89,7 +89,7 @@ public EncryptingPartEntity(final OutputStream cipherStream,
 
     @Override
     public boolean isRepeatable() {
-        return wrapped.isRepeatable();
+        return false;
```

Comment:
What does `isRepeatable` actually do?  If we never "repeat" how is the retry and snapshot code triggered?